src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -584,6 +584,22 @@ pub(crate) fn run_pass_manager(
     //      tools/lto/LTOCodeGenerator.cpp
     debug!("running the pass manager");
     unsafe {
+        if write::should_use_new_llvm_pass_manager(config) {
+            let opt_stage = if thin { llvm::OptStage::ThinLTO } else { llvm::OptStage::FatLTO };
+            let opt_level = config.opt_level.unwrap_or(config::OptLevel::No);
+            // See comment below for why this is necessary.
+            let opt_level = if let config::OptLevel::No = opt_level {
+                config::OptLevel::Less
+            } else {
+                opt_level
+            };
+            cgcx.prof.extra_verbose_generic_activity("LTO_passes").run(|| {
```

Comment:
Is "LTO_passes" the right name here?

Reply:
I don't know if it's the right name, it's just the existing name. See L654 below.

Reply:
Lets just name it "new_llvm_passes" or whatever so that’s it not the same thing as LTO stuff in profiles and whatnot.

Reply:
These are the LTO passes though?

Reply:
Oh well. Ignore me then.